client/tests.json

```diff
@@ -46,8 +67,22 @@
 			"plugins": {
 				"contact-form": {
 					"test": [ "shortcode", "validations" ]
+				},
+				"media": {
+					"restrict-size": {
+						"test": [ "index" ]
+					}
+				},
+				"wpcom-sourcecode": {
+					"test": [ "plugin" ]
 				}
 			}
+		},
+		"token-field": {
```

Comment:
Token fields are very slow comparing to other test suites:
![screen shot 2016-03-29 at 09 39 18](https://cloud.githubusercontent.com/assets/699132/14101061/26e24290-f592-11e5-84f0-7509dbf52734.png)